bench/src/main/scala/Benchmarks.scala

```diff
@@ -54,9 +55,20 @@ object Bench {
   def removeCompileOptions: Unit = new File(COMPILE_OPTS_FILE).delete()
 
   def storeCompileOptions(args: Array[String]): Unit = {
+    val standard_libs = System.getProperty("BENCH_CLASS_PATH")
+    val compiler_libs = System.getProperty("BENCH_COMPILER_CLASS_PATH")
+
+    val libs = if (args.contains("-with-compiler")) compiler_libs else standard_libs
```

Comment:
Is `-with-compiler` used anywhere? Where would it be used?
If this is used, we should test it in the `project/scripts/cmdTests`.

Reply:
Good idea, it'll used in the benchmarks to test "dotty". I'll add it to `project/scripts/cmdTests`.